app/assets/javascript/playerlistui.js

```diff
@@ -181,8 +181,13 @@ $(function() {
     webclientUI.players.count = $("#playercount");
 
     $("#player-filter").on("input", function() {
+        if (/^\s*$/.test($("#player-filter").val())) { /* Don't search when the string is blank */
+            return;
```

Comment:
actually no,  the setFilter should still be called even if to just reset the filter to blank.